CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 * In [text_columns()](https://py-pdf.github.io/fpdf2/extColumns.html), paragraph top/bottom margins didn't correctly trigger column breaks; [issue #1214](https://github.com/py-pdf/fpdf2/issues/1214)
 * [`fpdf.drawing.color_from_hex_string`](https://py-pdf.github.io/fpdf2/fpdf/drawing.html#fpdf.drawing.color_from_hex_string) did not test or mention accepting lowercase hex values.
 * handling of bidirectional text on `FPDF.get_string_width()` [issue #1231]
+* new translation of the tutorial in [Indonesia](https://py-pdf.github.io/fpdf2/Tutorial-id.html) - thanks to @odhyp
```

Comment:
The additions look good.
Just a spelling nitpick: In English, the name of the country is "Indonesia", but the language is specified with the adjective form "Indonesia**n**" (with an "n" at the end).
Once you fix the 4 places where this turns up, your translation is ready to be merged.

Reply:
Thanks for the feedback! I've corrected the instances where "Indonesia" was used and replaced them with "Indonesian" and the changes have been pushed.